### ISSUE TITLE
Fix Custom User Agent as lib / interceptor issue

### ISSUE THREAD

**Issue by Riztard** [keiyoushi/tachiyomi-extensions-source#3541]
### Describe why this should be added

when using custom user agent or using the random one from the "Custom User Agent" library, it will cause more issue if it got blocked by Cloudflare captcha. because when opening the webview to solve the captcha, the webview is not using the custom /random user agent, instead theyre using tachi UA from advanced setting and will got blocked again if user tries to use the source even if captcha is completed in webview. this will render solving captcha in webview useless.

need some workaround to pass the custom/random user agent to the webview or change back to the old way instead 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/mihonapp/mihon/issues/new/choose).
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.
